src/main/java/core/basesyntax/Animal.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Animal {
+
```

Comment:
let's add some state to our abstract class, that would me common for all animals to use benefits of inheritance(age/name...)

Reply:
added